modules/exoplanets/tasks/exoplanetsdatapreparationtask.cpp

```diff
@@ -423,17 +465,21 @@ glm::vec3 ExoplanetsDataPreparationTask::starPosition(const std::string& starNam
     return position;
 }
 
-float ExoplanetsDataPreparationTask::bvFromTeff(float teff) {
+float ExoplanetsDataPreparationTask::bvFromTeff(float teff,
+                                              const std::filesystem::path& conversionFile)
+{
     if (std::isnan(teff)) {
         return std::numeric_limits<float>::quiet_NaN();
     }
 
-    std::ifstream teffToBvFile(_teffToBvFilePath);
+    std::ifstream teffToBvFile(conversionFile);
     if (!teffToBvFile.good()) {
-        LERROR(fmt::format("Failed to open teff_bv.txt file"));
+        LERROR(fmt::format("Failed to open file '{}'", conversionFile));
```

Comment:
```suggestion
        LERROR(fmt::format("Failed to open file {}", conversionFile));
```

passing a `std::filesystem::path` will already add "..." around it